<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>translator</class>
 <widget class="QMainWindow" name="translator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>üì§‰∏ä‰º†Êñá‰ª∂</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QLabel" name="fileslabel">
                   <property name="text">
                    <string>Êñá‰ª∂</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QListWidget" name="listWidget"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="originallabel">
                 <property name="text">
                  <string>Ê∫êÊñáÊú¨</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="originaltext">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="translatedlabel">
                 <property name="text">
                  <string>ÁøªËØë</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="translatedtext">
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2,0,2,0,3,0,2,4">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QComboBox" name="sourcelang">
                 <property name="styleSheet">
                  <string notr="true">QComboBox {
    height: 30px;
}
</string>
                 </property>
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                 <property name="placeholderText">
                  <string> ÂéüÂßãËØ≠Ë®Ä</string>
                 </property>
                 <property name="duplicatesEnabled">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>5</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="targetlang">
                 <property name="styleSheet">
                  <string notr="true">QComboBox {
    height: 30px;
}
</string>
                 </property>
                 <property name="placeholderText">
                  <string>ÁõÆÊ†áËØ≠Ë®Ä</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>ÁøªËØëÊ®°Âºè</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="model1">
                   <property name="text">
                    <string>Âπ≥Ë°°</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">modelgroup</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="model2">
                   <property name="text">
                    <string>Âø´ÈÄü</string>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">modelgroup</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="model3">
                   <property name="text">
                    <string>Á≤æÁ°Æ</string>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">modelgroup</string>
                   </attribute>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>25</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="translatebutton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    height: 30px;
}

</string>
                 </property>
                 <property name="text">
                  <string>ÂºÄÂßãÁøªËØë</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,3,2,2">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>17</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>ÁøªËØëËøõÂ∫¶</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="progressBar">
                   <property name="value">
                    <number>24</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    color: gray; /* ËÆæÁΩÆÂ≠ó‰ΩìÈ¢úËâ≤‰∏∫ÁÅ∞Ëâ≤ */
}
</string>
                   </property>
                   <property name="text">
                    <string>Ââ©‰ΩôÊó∂Èó¥Ôºö2ÂàÜÈíü</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="styleSheet">
                    <string notr="true">QLabel {
    color: gray; /* ËÆæÁΩÆÂ≠ó‰ΩìÈ¢úËâ≤‰∏∫ÁÅ∞Ëâ≤ */
}
</string>
                   </property>
                   <property name="text">
                    <string>ËøõÂ∫¶Ôºö60/100Âè•</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>ÂéÜÂè≤È°µÈù¢ Â∞öÊú™ÂºÄÂèë</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page3">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>ËÆæÁΩÆÈ°µÈù¢ Â∞öÊú™ÂºÄÂèë</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page4">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Â∏ÆÂä©È°µÈù¢ Â∞öÊú™ÂºÄÂèë</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1126</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>‰∏ªÈ°µÈù¢</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu2">
    <property name="title">
     <string>ËÆæÁΩÆ</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu3">
    <property name="title">
     <string>Â∏ÆÂä©</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu4">
    <property name="title">
     <string>ÂéÜÂè≤</string>
    </property>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menu4"/>
   <addaction name="menu2"/>
   <addaction name="menu3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="modelgroup"/>
 </buttongroups>
</ui>
